components/splash/splash_bussinesses_using.js

```diff
@@ -33,7 +33,7 @@ const SplashBussinessesUsing = () => (
       justifyContent='space-around'
     >
       {logos.map((item, i) => (
-        <Link href={item.url} target='_blank' key={i}>
+        <Link href={item.url} target='_blank' rel='noopener noreferrer' key={i}>
```

Comment:
standard now checks for this, which is great bc this is actually a serious security risk